packages/react-native/src/Authenticator/common/DefaultContent/DefaultContent.tsx

```diff
@@ -46,10 +46,14 @@ export default function DefaultContent<
     <>
       {headerText ? (
         <Header style={themedStyles.header}>{headerText}</Header>
-      ) : (
-        <Header />
-      )}
-      {body ? body : null}
+      ) : null}
```

Comment:
Not sure why we would want to display an empty <Header />  if no header text is passed, changed to null, this way our existing tests fail if any of our subcomponents are missing header text.

Reply:
For context, not all our subcomponents were passing `headerText` previous to this PR, but there was always the potential that a user could have passed in a `Header`, so we would render the `Header` regardless. Now that we are requiring `headerText` for all subcomponents, think we can remove the ternary altogether